must-gather/collection-scripts/gather_metrics

```diff
@@ -20,7 +20,7 @@ oc exec -n openshift-monitoring prometheus-k8s-0 -- \
   > "${object_collection_path}/prometheus_target_metadata.json"
 
 # Prometheus - filtered metrics json using query_range and HTML files with charts
-for metric_name in node_load1 
+for metric_name in node_load1 velero_backup_total velero_backup_attempt_total velero_backup_success_total velero_backup_partial_failure_total velero_backup_failure_total velero_restore_total velero_restore_attempt_total velero_restore_success_total velero_restore_partial_failure_total velero_restore_failed_total velero_volume_snapshot_attempt_total velero_volume_snapshot_success_total velero_volume_snapshot_failure_total
```

Comment:
@savitharaghunathan we are only using these two metrics "velero_backup_total"
"velero_restore_total" , please remove the rest.

Reply:
Done. Thanks @shubham-pampattiwar :)